lib/compiler-llvm/src/abi/aarch64_systemv.rs

```diff
@@ -21,19 +21,14 @@ impl Abi for Aarch64SystemV {
     // Given a function definition, retrieve the parameter that is the vmctx pointer.
     fn get_vmctx_ptr_param<'ctx>(&self, func_value: &FunctionValue<'ctx>) -> PointerValue<'ctx> {
         func_value
-            .get_nth_param(
```

Comment:
I assume this is it a clippy lint?

The old code was much more readable, should use allow instead.

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3487/commits/ddc7628b7f6fd97318abb400eed0d1ecb6a78990